index.html

```diff
@@ -766,13 +760,7 @@ <h1>Alerts</h1>
                 </ul>
             </div>
 
-            <div class="alert expandable-alert alert-danger" role="alert">
-                <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-                    <span aria-hidden="true"><i class="material-icons">close</i></span>
-                </button>
-                <button type="button" class="read-more btn-link" data-toggle="collapse" data-target="#collapseDanger" aria-expanded="false" aria-controls="collapseDanger">
-                    Read more
-                </button>
+            <div class="alert expandable-alert alert-warning" role="alert">
```

Comment:
I think we should keep the `alert-danger` style in CSS for backwards compatibility

Reply:
What do you mean ? We got it but it's refactored, see :

![image](https://user-images.githubusercontent.com/14963751/71899991-2de62b00-315d-11ea-8131-e87f51ac5b73.png)

Reply:
@eternoendless https://github.com/PrestaShop/prestashop-ui-kit/pull/75/files#diff-eacf331f0ffc35d4b482f1d15a887d3bL761

Reply:
My mistake, I thought it was gone.